src/test/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/genericwhitespace/InputGenericWhitespaceDefault.java

```diff
@@ -84,13 +84,13 @@ public void beforeAndAfter() {
         Map<Class<?>, Integer> b = (Map<Class<?>, Integer>) a;
     }
     Object ok = new <String>Object();
-    Object notOkStart = new<String>Object(); // violation
-    Object notOkEnd = new <String> Object(); // violation
+    Object notOkStart = new<String>Object(); // violation ''<' is not preceded with whitespace.'
+    Object notOkEnd = new <String> Object(); // violation ''>' is followed by whitespace.'
     Object notOkStartAndEnd = new<String> Object(); // 2 violations
     Object okWithPackage = new <String>java.lang.Object();
     Object ok2 = new <String>Outer.Inner();
-    Object notOkStart2 = new<String>Outer.Inner(); // violation
-    Object notOkEnd2 = new <String> Outer.Inner(); // violation
+    Object notOkStart2 = new<String>Outer.Inner(); // violation ''<' is not preceded .*'
```

Comment:
keep doing.

Reply:
> let make it simple: all lines should have full message

Done, now all of them have a full message